src/notifications/use-push-notifications.tsx

```diff
@@ -53,23 +55,34 @@ export const usePushNotifications = () => {
   }, []);
 
   const register = useCallback(async () => {
+    try {
+      const token = await messaging().getToken();
+      if (!token) return;
+      mutateRegister({token, language: language});
+      return token;
+    } catch (e) {
+      Bugsnag.notify(`Failed to register for push notifications: ${e}`);
+    }
+  }, [language, mutateRegister]);
+
+  const requestPermissions = useCallback(async () => {
     setStatus('updating');
     const permissionStatus = await requestUserPermission();
-    const token = await messaging().getToken();
+    const token = await register();
     if (!token) {
       setStatus('error');
       return;
     }
-    registerMutation.mutate({token, language});
     setStatus(permissionStatus);
     return token;
-  }, [language, registerMutation]);
+  }, [register]);
 
   return {
     status,
     config: configQuery.data,
     updateConfig: configMutation.mutate,
     checkPermissions,
+    requestPermissions,
     register,
```

Comment:
`requestPermissions` and `register` returns the token, but is the token necessary to expose outside of the notifications module?

Reply:
That was added to make it visible in the debug menu, which has been used for testing: https://github.com/AtB-AS/mittatb-app/pull/4038